File: leetcode3/황은지/4020. Elevator Requests I.js
/**
 * @param {number} n
 * @param {number[]} requests
 * @return {number}
 */
var elevatorRequests = function (n, requests) {
  let seconds = 0;

  for (let i = 0; i < requests.length; i++) {
    if (i !== 0) seconds += Math.abs(requests[i] - requests[i - 1]);
    else seconds += requests[i];
  }

  return seconds;
};

File: leetcode3/이진희/4020. Elevator Requests I.java
/*

1. 아이디어 : 각 배열 숫자의 차이의 절댓값 더하기 Math.abs(requests[i]-requests[i-1]) + requests[0] (첫번째 층까지 올라가는 층수)

2. 시간복잡도 : O(N)

3. 자료구조/알고리즘 : 완전탐색

 */

class Solution {
    public int elevatorRequests(int n, int[] requests) {
        int ans = 0;
        for(int i=1; i<requests.length; i++) ans+=Math.abs(requests[i]-requests[i-1]);

        return ans+requests[0];
    }
}